spark/src/jmh/java/org/apache/iceberg/spark/source/IcebergSourceBenchmark.java

```diff
@@ -92,15 +93,24 @@ protected void cleanupFiles() throws IOException {
     }
   }
 
-  protected void setupSpark() {
-    spark = SparkSession.builder()
-        .config("spark.ui.enabled", false)
-        .master("local")
-        .getOrCreate();
+  protected void setupSpark(boolean enableDictionaryEncoding) {
+    SparkSession.Builder builder = SparkSession.builder()
+            .config("spark.ui.enabled", false);
```

Comment:
Does this make Spark start faster? Should we use it in other tests?

Reply:
I am actually not sure. But it does look like a lot of tests within Spark turn off the UI. 

For ex- https://github.com/apache/spark/blob/a222644e1df907d0aba19634a166e146dfb4f551/core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala#L268